### PR TITLE
Prep changelog for release

### PR DESCRIPTION
**Pull request by zangell44**
Add release notes for 0.3.5 and 0.3.4.

### Checklist

- [ ] References any related issue by including "Closes #<Issue Number>" or "Closes <Issue URL>".
  - If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect-kubernetes/issues/new/choose) first.
- [ ] Includes tests or only affects documentation.
- [ ] Passes `pre-commit` checks.
  - Run `pre-commit install && pre-commit run --all` locally for formatting and linting.
- [ ] Includes screenshots of documentation updates.
  - Run `mkdocs serve` view documentation locally.
- [ ] Summarizes PR's changes in [CHANGELOG.md](https://github.com/PrefectHQ/prefect-kubernetes/blob/main/CHANGELOG.md)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,25 +9,36 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 
 ### Fixed
 
+### Security
+
+## 0.3.5
+
+### Changed
+
+- Update branding
```
Comment (zzstoatzz) [inline CHANGELOG.md:26]:
```suggestion
- Update branding [#110](https://github.com/PrefectHQ/prefect-kubernetes/pull/110)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,25 +9,36 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+
+### Fixed
+
 - Added Kubernetes client caching to prevent worker from running out of sockets - [#111](https://github.com/PrefectHQ/prefect-kubernetes/pull/111)
 
+## 0.3.4
+
```
Comment (zzstoatzz) [inline CHANGELOG.md:33]:
```suggestion

Released February 1st, 2024.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -9,25 +9,36 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 ### Deprecated
 
 ### Removed
 
 ### Fixed
 
+### Security
+
+## 0.3.5
+
```
Comment (zzstoatzz) [inline CHANGELOG.md:23]:
```suggestion

Released February 8th, 2024.

```

**Review by zzstoatzz — APPROVED** [review]
looks like we forgot the release date last time but otherwise lgtm!

**Comment by zangell44** [comment]
Thanks for the fixes @zzstoatzz!

**Pull request merged**